### DISCUSSION TITLE
We are unable to complete your signup.

### DISCUSSION THREAD

**Discussion by tonkolviktor** [vercel/community#4256] [Account Issue] [answered]
### Question

Why? just why? I see that everyone is opening tickets like this, is this really the best way? Am I supposed to post my email here?

**Comment by tonkolviktor** [comment]
please do not post this page: https://vercel.com/guides/why-can-i-not-signup
it's 100% useless

**Reply by amyegan** [reply, marked as answer]
Sorry you're having trouble with registration. Please check your email for a message with instructions on next steps.

> In certain situations, our system may email you to share more details about the error and outline next steps. If you do not receive an email which outlines next steps, we will be unable to discuss the error further.

We can't safely discuss account information in public because of security concerns, but you can find a list of direct channels to contact Vercel Support here: https://vercel.com/guides/how-to-get-vercel-support#direct-channels

**Reply by tonkolviktor** [reply]
thanks for the answer Amy these emails are already much better.
At the same time the problem is still not solved and I still cannot log in.

I do not understand why it's not linked to the other page

**Reply by tonkolviktor** [reply]
btw still does not work after 2 days and 2 emails to the support....

**Reply by amyegan** [reply]
Tickets are processed by the order in which they are received. That means you will need to wait for other requests submitted by customers on the same plan as you to be fulfilled before the team can respond to yours. 

I know this is a frustrating situation, and we all appreciate your patience while the team works through a large number of cases. More info about response times and how Vercel Support works can be found at https://vercel.com/guides/vercel-support-queue-time

**Reply by tonkolviktor** [reply]
thank you @amyegan, well it still does not work, I mean....

**Reply by tonkolviktor** [reply]
in the last 15mins I've registered a new email address and a new github account (btw both of which worked without any issues)
vercel even with this brand new shiny account cannot make a sign up work...
![image](https://github.com/vercel/community/assets/4534324/b47fb834-4941-40fe-a09e-7737d4a5f17b)
based on what do you block an email which did not even exist 30mins ago?

**Reply by tonkolviktor** [reply]
Hi @leerob VP of Developer Experience, could you please reflect on the results of this search: https://github.com/orgs/vercel/discussions/categories/account-issue
Is that Developer Experience by design?
Thanks, Viktor

**Reply by amyegan** [reply]
I don't have the details available to me, but my guess is the fresh accounts may have caused this new signup to seem suspicious. I know this isn't a fun experience and our team is working to improve it. 

There is an unusually high number of cases for the team to work through, and we all appreciate your patience and understanding while the team addresses them as quickly as possible.

**Reply by tonkolviktor** [reply]
![image](https://github.com/vercel/community/assets/4534324/ae7dcaf3-12ac-4fdd-b132-9f6b3b3e6c44)

well now I have Failed to fetch, I'm unsure if that's a progress or not...

**Reply by tonkolviktor** [reply]
ohh well, failed to fatch was an additional issue, which got solved, sooooo we are back to unable to complete the signup. Now answer for more then a week now from the support.
I'm invited to a Pro subscription, I do not understand this.

Please someone explain how this process should work, thanks.

**Reply by tonkolviktor** [reply]
my whole hearted thanks goes to @DeWjjj
https://github.com/orgs/vercel/discussions/4110#discussioncomment-7057085

As it turns out one should not use VPN while signing up, because ??? (it's evil i guess)
Anyhow @amyegan would be nice to inform people about this "small" detail.
